cmd/clusterctl/client/config/providers_client.go

```diff
@@ -196,6 +197,11 @@ func (p *providersClient) defaults() []Provider {
 			url:          "https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/releases/latest/infrastructure-components.yaml",
 			providerType: clusterctlv1.InfrastructureProviderType,
 		},
+		&provider{
+			name:         CloudStackProviderName,
+			url:          "https://github.com/kubernetes-sigs/cluster-api-provider-cloudstack/releases/latest/infrastructure-components.yaml",
```

Comment:
LGTM, this is subject to donation https://github.com/kubernetes/org/issues/3279

Reply:
Repo migrated